packages/datadog_flutter_plugin/android/src/test/kotlin/com/datadoghq/flutter/DatadogSdkPluginTest.kt

```diff
@@ -157,6 +162,87 @@ class DatadogSdkPluginTest {
         verify(mockResult).success(null)
     }
 
+    @Test
+    fun `M not issue warning W initialize called with same configuration`(
+        forge: Forge
+    ) {
+        // GIVEN
+        mockkStatic(Log::class)
+        Datadog.setVerbosity(Log.INFO)
+
+        val config = mapOf(
+            "clientToken" to forge.anAlphaNumericalString(),
+            "env" to forge.anAlphabeticalString(),
+            "trackingConsent" to "TrackingConsent.granted",
+            "nativeCrashReportEnabled" to true,
+            "loggingConfiguration" to null
+        )
+        val methodCall = MethodCall(
+            "initialize",
+            mapOf(
+                "configuration" to config
+            )
+        )
+        val mockResult = mock<MethodChannel.Result>()
+        plugin.onMethodCall(methodCall, mockResult)
+
+        // WHEN
+        val methodCallB = MethodCall(
+            "initialize",
+            mapOf(
+                "configuration" to config
+            )
+        )
+        val mockResultB = mock<MethodChannel.Result>()
+        plugin.onMethodCall(methodCallB, mockResultB)
+
+        // THEN
+        io.mockk.verify(exactly = 0) { Log.println(any(), any(), any()) }
+    }
+
+    @Test
+    fun `M issue warning W initialize called with different configuration`(
+        forge: Forge
+    ) {
+        // GIVEN
+        Datadog.setVerbosity(Log.INFO)
+
+        val methodCall = MethodCall(
+            "initialize",
+            mapOf(
+                "configuration" to mapOf(
+                    "clientToken" to forge.anAlphaNumericalString(),
+                    "env" to forge.anAlphabeticalString(),
+                    "trackingConsent" to "TrackingConsent.granted",
+                    "nativeCrashReportEnabled" to true,
+                    "loggingConfiguration" to null
+                )
+            )
+        )
+        val mockResult = mock<MethodChannel.Result>()
+        plugin.onMethodCall(methodCall, mockResult)
+
+        // WHEN
+        mockkStatic(Log::class)
+        val methodCallB = MethodCall(
+            "initialize",
+            mapOf(
+                "configuration" to mapOf(
+                    "clientToken" to forge.anAlphaNumericalString(),
+                    "env" to forge.anAlphabeticalString(),
+                    "trackingConsent" to "TrackingConsent.granted",
+                    "nativeCrashReportEnabled" to true,
+                    "loggingConfiguration" to null
+                )
+            )
+        )
+        val mockResultB = mock<MethodChannel.Result>()
+        plugin.onMethodCall(methodCallB, mockResultB)
+
+        // THEN
+        io.mockk.verify(exactly = 1) { Log.e(any(), match { it.contains("🔥") }) }
```

Comment:
I think we can remove `any()` and `match()` calls here and reference directly tag and message, because they are `internal` and should be available.